src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -83,6 +88,10 @@ List only the disabled partitions.
 
 List all in json format.
   rpk cluster partition list -a --format json
+
+Show how the partitions are distributed across brokers.
+  rpk cluster partitions list -a --broker-distribution
+  rpk cluster partitions foo bar --broker-distribution
 `,
 		Run: func(cmd *cobra.Command, topics []string) {
 			f := p.Formatter
```

Comment:
Now the help text should return `resp` instead of `[]rpadmin.ClusterPartition{}`

Reply:
Great catch! Thank you.